spec/features/admin/view_enrollment_exemption_spec.rb

```diff
@@ -54,13 +54,13 @@
         end
 
         within "#secondary-contact-details" do
-          expect(page).to have_text(enrollment.secondary_contact.full_name)
+          expect(page).to have_text(enrollment.secondary_contact.email_address)
         end
 
         ee = enrollment.enrollment_exemptions.first
 
         within "#actions" do
-          expect(page).to have_css("#update-enrollment-exemption-status")
+          # expect(page).to have_css("#update-enrollment-exemption-status")
```

Comment:
Should we just remove this line entirely?

Reply:
yes!